docs/design/modules/engine/raycasting/algorithm.md

```
@@ -2,45 +2,84 @@
-// 擬似コードをここに記述
+for x = 0 to WINDOW_WIDTH:
+  camera_x = 2 * x / WINDOW_WIDTH - 1
+  ray_dir = player.dir + player.plane * camera_x
+
+  // DDA準備
+  map_x = int(player.pos_x)
+  map_y = int(player.pos_y)
+  delta_dist = abs(1 / ray_dir)
+
+  // ステップ方向と初期距離
+  if ray_dir.x < 0:
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)